editor/project_export.cpp

```diff
@@ -931,8 +931,17 @@ void ProjectExportDialog::_export_project() {
 		export_project->add_filter("*." + extension_list[i] + " ; " + platform->get_name() + " Export");
 	}
 
-	if (current->get_export_path() != "") {
-		export_project->set_current_path(current->get_export_path());
+	String current_preset_export_path = current->get_export_path();
+
+	if (current_preset_export_path != "") {
+
+		if (!DirAccess::exists(current_preset_export_path.get_base_dir())) {
+
+			DirAccess *da = DirAccess::create(DirAccess::ACCESS_FILESYSTEM);
```

Comment:
Should also be `DirAccessRef` here. Please also squash the commits together.

Reply:
Commits squashed. DirAccess removed there in favor of DirAccessRef.